src/main/java/com/arpnetworking/tsdcore/sinks/WavefrontSink.java

```diff
@@ -274,11 +280,13 @@ public Builder setIdFile(final String value) {
         }
 
         /**
-         * Set of dimension names to be passed through as a Wavefront annotation/tag.
+         * No longer used. All dimensions are passed to wavefront. Restriction of dimensions should be done elsewhere.
          *
+         * @deprecated All dimensions are passed to Wavefront.
          * @param value Set of white listed dimension names
          * @return This instance of <code>Builder</code>
          */
+        @Deprecated
```

Comment:
This is not backwards compatible. Why not make it optional and allow all if not set?